src/App.js

```diff
@@ -1,25 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Button from "./components/Button";
+import Footer from "./components/Footer";
+//import Footer from "./components/Footer/Footer";
+//import AuctionItem from "./components/AuctionItem";
+import IntroductionAndConclusion from "./components/IntroductionAndConclusion";
+import "./index.css";
 
-function App() {
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <IntroductionAndConclusion
+        title={"Title"}
+        buttonName={"CTA TEXT"}
+        text={
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris."
+        }
+      ></IntroductionAndConclusion>
+      <IntroductionAndConclusion
+        title={"Sell with Softly"}
+        buttonName={"START"}
+        text={
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris."
+        }
+        img={"./images/Rectangle 14.jpg"}
+      ></IntroductionAndConclusion>
+      <div style={{ margin: "5px" }}>
+        <Button>BID</Button>
+      </div>
+      <div style={{ margin: "5px" }}>
+        <Button styleForButton={"secondary"}>OUT BID</Button>
+      </div>
+      <div style={{ margin: "5px" }}>
+        <Button styleForButton={"seeMore"} icon={"./images/Polygon1.svg"}>
+          See More
+        </Button>
+      </div>
+      <div style={{ margin: "5px" }}>
+        <Button styleForButton={"loadMore"} icon={"./images/Polygon1.svg"}>
+          Load More
+        </Button>
+      </div>
+      <div style={{ margin: "5px" }}>
+        <Button styleForButton={"sortBy"} icon={"./images/Polygon1.svg"}>
+          Sort By
+        </Button>
+      </div>
+      <Footer></Footer>
```

Comment:
When you use a HTML element or React element that doesn't have children, you can insert it as an empty element like so `<Footer />, <div />...`